flickit-assessment-common/src/main/resources/i18n/kit/messages_en.properties

```diff
@@ -58,3 +58,18 @@ create-kit-by-dsl.isPrivate.notNull='isPrivate' may not be null
 create-kit-by-dsl.kit-dsl-json-id.notNull='kitJsonDslId' may not be null
 create-kit-by-dsl.expert-group-id.notNull='expertGroupId' may not be null
 create-kit-by-dsl.tag-ids.notNull='tagIds' may not be null
+
+create-expert-group.title.notBlank='title' may not be empty
+create-expert-group.title.size.min='title' must be more or equal to 3
+create-expert-group.title.size.max='title' must be less than or equal to 100
+create-expert-group.bio.notBlank='bio' may not be empty
+create-expert-group.bio.size.min='bio' must be more than or equal to 3
+create-expert-group.bio.size.max='bio' must be less than or equal to 200
+create-expert-group.picture.file-format='picture' should be valid file format
```

Comment:
`should be` or `should have a`?

what about this?
'pucture' file should have a valid format such as .png, .jpeg, .jpg